docs/3.0/common/reloadable.md

```diff
@@ -0,0 +1,17 @@
+## Reloadable
+
+Starting from this version (I don't know), Avo introduces the option to enable a `reloadable` feature for association tables. This feature allows users to manually refresh the content of association table.
```

Comment:
You'll find the version number in `lib/avo/version.rb` or from https://github.com/avo-hq/avo/releases (check where the feature was added).

Instead of starting with "Starting from this version...", Just start by saying what the feature does. "The `reoadable` option ...."

For version requirements, please use the `<VersionReq version="2.28" />` component under the title (with one line between them) like [here](https://github.com/avo-hq/avodocs/blob/main/docs/3.0/authorization.md#L302).

I would say something along the lines of "The `reloadable` option adds a reload icon next to the association title so users can easily reload just that turbo-frame instead of doing a full page reload"

Reply:
Thanks for the suggestions.